src/libraries/System.Net.Mail/tests/Unit/System.Net.Mail.Unit.Tests.csproj

```diff
@@ -19,7 +18,7 @@
     <Compile Include="MessageTests\MessageHeaderBehaviorTest.cs" />
     <Compile Include="MessageTests\ReplyToListTest.cs" />
     <Compile Include="QuotedPrintableStreamTest.cs" />
-    <Compile Include="SmtpConnectionTests\EhloParseExtensionsTest.cs" />
+    <Compile Include="SmtpConnectionTests\EhloParseExtensionsTest.cs" Condition="'$(TargetsBrowser)' != 'true'" />
```

Comment:
We're skipping this test with an attribute, is this needed?

Reply:
It is, SmtpConnection is internal and marking it in both places makes it a little more readable.